graylog2-server/src/main/java/org/graylog/plugins/pipelineprocessor/db/mongodb/PaginatedMongoDbPipelineService.java

```diff
@@ -0,0 +1,35 @@
+package org.graylog.plugins.pipelineprocessor.db.mongodb;
+
+import org.graylog.plugins.pipelineprocessor.db.PaginatedPipelineService;
+import org.graylog.plugins.pipelineprocessor.db.PipelineDao;
+import org.graylog2.bindings.providers.MongoJackObjectMapperProvider;
+import org.graylog2.database.MongoConnection;
+import org.graylog2.database.PaginatedDbService;
+import org.graylog2.database.PaginatedList;
+import org.graylog2.search.SearchQuery;
+import org.mongojack.DBQuery;
+import org.mongojack.DBSort;
+
+import javax.inject.Inject;
+import java.util.function.Predicate;
+
+public class PaginatedMongoDbPipelineService extends PaginatedDbService<PipelineDao> implements PaginatedPipelineService {
+    private static final String COLLECTION_NAME = "pipeline_processor_pipelines";
+
+    @Inject
+    public PaginatedMongoDbPipelineService(MongoConnection mongoConnection,
+                                           MongoJackObjectMapperProvider mapper) {
+        super(mongoConnection, mapper, PipelineDao.class, COLLECTION_NAME);
+    }
+
+    public long count() {
+        return db.count();
+    }
```

Comment:
I was taking a look, and it seems the `count` method is not used anywhere anymore. It looks like it was introduced in the initial implementation (https://github.com/Graylog2/graylog2-server/commit/33d0fe06d5e99d887c2b49d71f483e782b45b8ef), but apparently the `PipelinesResource` logic no longer uses it. I know it existed before, but I think it would be good to remove it in this PR as well, since it's not used and not exposed on the interface. What do you think?